project/script_VMR.tcl

```diff
@@ -22,8 +22,7 @@ source settings_hls.tcl
 
 # data files
 add_files -tb ../emData/VMR/tables/
```

Comment:
I believe this line can also be removed.

Reply:
removed!